src/StackAdmin/AzureRM/AzureRM.psd1

```diff
@@ -1,100 +1,103 @@
-﻿#  
```

Comment:
@solankisamir please revert the changes made in this file, both of the `AzureRM.psm1` files, and `AzureStack.psm1`